### PR TITLE
url.md: change comment documentation to verified

### PR DESCRIPTION
**Pull request by vitorgalvao**
Waiting on https://github.com/Homebrew/brew/pull/7772.

Ping @claui.

### CODE REVIEW

File: doc/cask_language_reference/stanzas/url.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ When a plain URL string is insufficient to fetch a file, additional information
 
 | key                | value       |
 | ------------------ | ----------- |
+| `verified:`        | a string repeating the beginning of `url`, for verification purposes. [See below](#when-url-and-homepage-hostnames-differ-add-verified).
 | `using:`           | the symbol `:post` is the only legal value
 | `cookies:`         | a hash of cookies to be set in the download request
 | `referer:`         | a string holding the URL to set as referrer in the download request
```
Comment (miccal) [inline doc/cask_language_reference/stanzas/url.md:16]: `referrer` -> `referer`

Comment (vitorgalvao) [reply doc/cask_language_reference/stanzas/url.md:16]: No, it’s [`referer`](https://en.wikipedia.org/wiki/HTTP_referer). I’ve also wondered about that in the past. Either way, that’s how the code is, so if it were misspelled we’d need to fix more than the documentation.

Comment (vitorgalvao) [reply doc/cask_language_reference/stanzas/url.md:16]: Oh, I see what you mean.

Comment (vitorgalvao) [reply doc/cask_language_reference/stanzas/url.md:16]: https://github.com/Homebrew/homebrew-cask/pull/84674

**Comment by miccal** [comment]
I have added some `style` fixes.

**Review by miccal — APPROVED** [review]

**Pull request merged**